shared/src/main/scala/effekt/generator/JavaScript.scala

```diff
@@ -166,7 +167,7 @@ trait JavaScriptBase extends ParenPrettyPrinter {
     case _: symbols.Effect   => toDoc(id.name)
     case _: symbols.EffectOp => "op$" + id.name.toString
     case _ => id.name match {
-      case name: QualifiedName if name.module != C.module => link(name, s"${name.module.name}.${name.name}")
+      case name: NestedName if name.parent != C.module.name => link(name, s"${name.parent.localName}.${name.localName}")
```

Comment:
Not sure if this still works the way it should.

Reply:
The point of this check is that names in the same module should not be referred to by the full qualified name, but the local name should be used instead.

Reply:
Without having checked it explicitly, your refactoring looks about right here.